CIME/build.py

```diff
@@ -211,7 +211,7 @@ def generate_makefile_macro(case, caseroot):
         )
         fd.write(all_output)
 
-
+# pylint:disable=shared_lib
```

Comment:
I think that this is wrong.  It should be something like 
disable=unused-argument

Reply:
Ah I am just gonna ask you why this does not work. Thanks and commit the fix!